index.html

```diff
@@ -404,6 +403,63 @@ <h2>
         handler event type</a> is <code>"change"</code>.
       </p>
     </section>
+    <section>
+      <h2>
+        Extensions to the <a>document</a> interface
+      </h2>
+      <section>
+        <h2>
+          Internal Slots
+        </h2>
+        <table>
+          <thead>
+            <tr>
+              <th>
+                Internal Slot
+              </th>
+              <th>
+                Description
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <dfn>[[\orientationLock]]</dfn>
+              </td>
+              <td>
+                The <a>[[\orientationLock]]</a> represents a <a>document</a>'s
+                <a>orientation lock</a> as an <a data-cite=
+                "infra#ordered-set">unordered set</a> of
+                <a>orientationType</a>.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn>[[\defaultOrientation]]</dfn>
+              </td>
+              <td>
+                The set of orientations to which the screen orientation is
```

Comment:
We need to check the "set" is ordered or unordered. If the set is ordered, then it's a "list":
https://infra.spec.whatwg.org/#list

Reply:
Remember, key thing about internal slots is that they always have an actual data type (list, unordered set, Promise, etc.).